### DISCUSSION TITLE
请问如何永久修改public文件夹下的index.html内容？

### DISCUSSION THREAD

**Discussion by Korltex** [hexojs/hexo#5439] [Question / Answer] [answered]
我希望改变tag界面的一些字体样式，所以我修改了public/tags/index.html里的相关代码，然后hexo -d，push到仓库。可是下一次hexo -g重新生成静态文件时，相关修改会恢复原状。请问该如何解决这个问题？

**Comment by uiolee** [comment]
`public/` is generated by `hexo` and you should not modify it. 

You should modify the `theme` if you need to

**Reply by Korltex** [reply, marked as answer]
Thank you for responsing！I modified `Butterfly`(theme I use)  settings and achieved my goal.
